__tests__/app.test.js

```diff
@@ -236,3 +236,46 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  it("200: PATCH - responds with updated article with votes changed depending on the request given", () => {
+    const update = { inc_votes: -50 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(update)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toHaveProperty("author", expect.any(String));
+        expect(article).toHaveProperty("title", expect.any(String));
+        expect(article).toHaveProperty("article_id", 1);
+        expect(article).toHaveProperty("topic", expect.any(String));
+        expect(article).toHaveProperty("created_at", expect.any(String));
+        expect(article).toHaveProperty("votes", 50);
+        expect(article).toHaveProperty("article_img_url", expect.any(String));
+        expect(article).toHaveProperty("body", expect.any(String));
+      });
+  });
+  it("400: PATCH -  responds with 400 status code passed a string for votes", () => {
+    const update = { inc_votes: "hello" };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(update)
+      .expect(400)
+      .then(({ body }) => {
+        const error = body.message;
+        expect(error).toBe("Bad Request");
+      });
+  });
+  it("404: PATCH -  responds with 404 status code when passed an article_id which doesn't exist", () => {
+    const update = { inc_votes: 50 };
+    return request(app)
+      .patch("/api/articles/1000")
+      .send(update)
+      .expect(404)
+      .then(({ body }) => {
+        const error = body.message;
+        expect(error).toBe("Article not found");
+      });
+  });
+});
```

Comment:
Could we just add one more test for a 400 here please? We need to test what happens if we have a missing/incorrect body (i.e. inc_votes is not a number, or is missing)